packages/gestalt/src/Tabs.js

```diff
@@ -25,54 +52,68 @@ export default function Tabs({
     text: React.Node,
   |}>,
   wrap?: boolean,
-|}): React.Node {
-  const handleTabClick = (i: number, e: SyntheticMouseEvent<>) =>
-    onChange({ activeTabIndex: i, event: e });
+|};
 
+export default function Tabs({
+  activeTabIndex,
+  onChange,
+  size = 'md',
+  tabs,
+  wrap,
+}: Props): React.Node {
   return (
-    <div
-      className={classnames(
-        styles.Tabs,
-        wrap && layout.flexWrap,
-        size === 'md' ? layout.medium : layout.large
-      )}
+    <Box
+      alignItems="center"
+      display="flex"
+      marginEnd={-1}
+      marginStart={-1}
+      minHeight={size === 'lg' ? 48 : 40}
```

Comment:
@jackhsu978 something odd happened here where the minHeight does change but the size of the Tab itself does not.
![image](https://user-images.githubusercontent.com/127199/86024380-c6a98500-b9e1-11ea-808b-2e3b76aa44d6.png)

Reply:
fixed. the `size` now not only controls the `minHeight`, but also the text font size. I think it makes more sense that, when the tab component is taller, the text inside of the tab component is also bigger.